src/current/_data/releases.yml

```diff
@@ -7460,4 +7460,31 @@
     docker_arm_experimental: false
     docker_arm_limited_access: false
   source: true
-  previous_release: v24.3.0-beta.3
\ No newline at end of file
+  previous_release: v24.3.0-beta.3
+
+- release_name: v24.2.6
+  major_version: v24.2
+  release_date: '2024-12-12'
+  release_type: Production
+  go_version: go1.22.8
+  sha: 74f60821a82c570ab58b8d7716c1d74b85e87df0
+  has_sql_only: true
+  has_sha256sum: true
+  mac:
+    mac_arm: true
+    mac_arm_experimental: true
+    mac_arm_limited_access: false
+  windows: true
+  linux:
+    linux_arm: true
+    linux_arm_experimental: false
+    linux_arm_limited_access: false
+    linux_intel_fips: true
+    linux_arm_fips: false
+  docker:
+    docker_image: cockroachdb/cockroach
+    docker_arm: true
+    docker_arm_experimental: false
+    docker_arm_limited_access: false
+  source: true
+  previous_release: v24.2.5
```

Comment:
i think this is missing the cloud-only YAML block from the '[Publish Release Notes](https://cockroachlabs.atlassian.net/wiki/spaces/ED/pages/1134166401/Publish+Release+Notes#Handle-a-Cloud-first-release)' wiki page ??  asking bc PR title says cloud first

Reply:
Whoops! Totally forgot that, thanks for catching! It's there now, phew!